packages/firestore/test/unit/specs/existence_filter_spec.test.ts

```diff
@@ -631,4 +631,189 @@ describeSpec('Existence Filters:', [], () => {
         .expectLimboDocs(...docKeys.slice(50))
     );
   });
+
+  specTest('Bloom filter handles new document adds', [], () => {
```

Comment:
This test name "Bloom filter handles new document adds" doesn't seem to match what the test is testing. The test appears to be testing _deleted_ documents.

Reply:
Also, this test "Bloom filter handles new document adds" looks very similar to "Full re-query is skipped when bloom filter can identify documents deleted" above, except that it uses 100 documents instead of 2 and doesn't check anything after the documents go into limbo. Is this test necessary?

Reply:
I forgot to delete this test case.